bugbug/tools/code_review.py

```diff
@@ -126,7 +126,7 @@ def __init__(self, patch_id) -> None:
 class Patch:
     raw_diff: str
 
-    def __init__(self, raw_diff) -> None:
+    def __init__(self, raw_diff, file_diff=None) -> None:
```

Comment:
I guess you forgot dropping it from the constructor:
```suggestion
    def __init__(self, raw_diff) -> None:
```